### DISCUSSION TITLE
Missing accessibility option of adding new blocks between others

### DISCUSSION THREAD

**Discussion by xXBuilderBXx** [GitbookIO/community#244] [Feature Requests]
GitBooks has recently made some changes this month that seems to have removed or broken the add new block divider when you hover over the empty space.

You used to be able to add new blocks in between other blocks and this is a big accessiblity issue because now i can't seem to add new blocks anywhere without having to go the end of a text block and pressing enter which does not work for adding blocks at the top of the page too.
I tried force refresh and clearing the cache but it's not working either.
![image](https://github.com/GitbookIO/community/assets/17956143/27fd8538-1cdb-43a3-8dce-11ef560f2dd4)

**Comment by taszap** [comment]
Hey @xXBuilderBXx 

So sorry you are having trouble! It was an intentional change - to add new block simply press Enter. As you pointed out in some blocks such as hints, code blocks you will have to hit CMD (or CTRL) + Enter to leave the block and create new line of text. 

Hope that helps!

**Reply by xXBuilderBXx** [reply]
Thanks for the ctrl tip although this isn't very good for accessibility or new users and being able to create docs is much slower without this hopefully it could be added back.

**Reply by SRyanPlanitar** [reply]
Agreed, it was useful to be able to click the "+" to add a block instead of trying to find the end of a block, it was actually a feature our team enjoyed over other document editing.

**Reply by WenKing6** [reply]
Me too

**Reply by jarnek73** [reply]
Please bring back the "+"... it's so much easier.

**Comment by xXBuilderBXx** [comment]
Looks like they added back the + option which is now to the side of elements, this will work out ok now :)
![image](https://github.com/GitbookIO/community/assets/17956143/fcb7c287-01eb-486d-b7dd-1674074ab4c6)
